calibrators/marker_radar_lidar_calibrator/src/marker_radar_lidar_calibrator.cpp

```diff
@@ -365,7 +404,7 @@ void ExtrinsicReflectorBasedCalibrator::backgroundModelRequestCallback(
     }
 
     RCLCPP_WARN_THROTTLE(
-      this->get_logger(), *this->get_clock(), 5000, "Waiting for the calibration to end");
+      this->get_logger(), *this->get_clock(), 5000, "Waiting for the extracting background model");
```

Comment:
Grammar

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/ba05cc4ce7421b28de27fb995607aed1de5c3516

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/12b356233707aa31d5be2cd629d19bbcef3401cb